### PR TITLE
update analysis with new features

### PR DESCRIPTION
**Pull request by andrealee011**
- update hypothesis test with new features
- change png table to csv and kable
- update final report with new features
- add motivation to final report
- strengthen conclusion in final report

### CODE REVIEW

File: doc/avocado_predictors_report.Rmd [hunk 1]
```diff
@@ -62,53 +71,55 @@ We wanted to determine which features might be the most important to include in
...
 
-# Results
-
 ## Hypothesis Test
 
-First, we conducted a hypothesis test to determine if any of the features are correlated to the target. To conduct a hypothesis test, we fitted an additive linear model and interpreted the p-values to determine which features are significant. We chose a significance level of 0.05 as it is the industry standard. We chose not to choose a stricter significance level (i.e. 0.01 or 0.001) as we do not believe that predicting avocado prices requires as conservative of a test. 
+Before undergoing our main analysis, we first conducted a hypothesis test to determine if any of the features are correlated to the target. To conduct a hypothesis test, we fitted an additive linear model and interpreted the p-values to determine which features are significant. We chose a significance level of 0.05 as it is the industry standard. We chose not to choose a stricter significance level (i.e. 0.01 or 0.001) as we do not believe that predicting avocado prices requires as conservative of a test.
```
Comment (techrah) [inline doc/avocado_predictors_report.Rmd:76]: > conducted a hypothesis test to determine if any of the features are correlated to the target.

I think we're only checking the significance of specific features, not any correlations.

**Review by techrah — CHANGES_REQUESTED** [review]

File: doc/avocado_predictors_report.Rmd (hunk 1 above)
Comment (katieb1) [reply doc/avocado_predictors_report.Rmd:76]: I think we are checking if they are significantly correlated

**Review by katieb1 — APPROVED** [review]
Looks good!

File: doc/avocado_predictors_report.Rmd (hunk 1 above)
Comment (techrah) [reply doc/avocado_predictors_report.Rmd:76]: Ah, correlated to the **target** ... I see. 👍

**Review by techrah — APPROVED** [review]

**Pull request merged**